sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/Generated/Models/ConversionErrorCode.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.MixedReality.ObjectAnchors.Conversion.Models
+{
+    /// <summary> The ConversionErrorCode. </summary>
+    public readonly partial struct ConversionErrorCode : IEquatable<ConversionErrorCode>
+    {
+        private readonly string _value;
+
+        /// <summary> Determines if two <see cref="ConversionErrorCode"/> values are the same. </summary>
```

Comment:
This description seems misplaced. /cc @ShivangiReja

Reply:
Agreed, but requires autogen fix